v-next.sql

```diff
@@ -669,3 +669,582 @@ AND NOT EXISTS (SELECT * FROM openchpl.cqm_criterion cc INNER JOIN openchpl.cqm_
 
 -- END OCD-2533
 ----------------------------------------------------
+
+-- OCD-2532 - 2015 functionality tested should be restricted by criteria
+-- 1) Create new table to support relationship between criteria and test functionality
+-- 2) Add triggers to new table
+-- 3) Rename old column
```

Comment:
This shouldn't be here either

Reply:
Just shoot me...  Made the change and didn't save it.